### DISCUSSION TITLE
Implementing Multivariate Logit Parameterization For Omega

### DISCUSSION THREAD

**Discussion by JonKnights** [nlmixrdevelopment/nlmixr#607] [Q&A]
Hi - I'm attempting to implement a multivariate logit parameterization (from Tsamandouras et al. below) of an Omega matrix. I pulled previous parameter min, max values and fixed those and am only trying to estimate the omega matrix leveraging SAEM algorithm... has anyone attempted a parameterization like this using NLMIXR? Of note, the authors provided parameterization of the NONMEM code which is what I was attempting to implement in the package, bu treceived the following error dumps on screen.

<img width="585" alt="Screen Shot 2022-04-25 at 10 51 06 AM" src="https://user-images.githubusercontent.com/81582526/165116505-9ebfb052-aecb-4fbd-a379-2ab7250a2353.png">

Tsamandouras N, Wendling T, Rostami-Hodjegan A, Galetin A, Aarons L. Incorporation of stochastic variability in mechanistic population pharmacokinetic models: handling the physiological constraints using normal transformations. J Pharmacokinet Pharmacodyn. 2015 Aug;42(4):349-73. doi: 10.1007/s10928-015-9418-0. Epub 2015 May 26. PMID: 26006250

**Comment by mattfidler** [comment]
I have implemented multivariate logit in the omega block using `nlmixr`, but I was using the approach in [10.1007/s11095-009-9931-1](https://doi.org/10.1007/s11095-009-9931-1)

In that case I fixed a population parameter to zero and let the eta vary.  This made a mu-referenced eta that didn't actually do anything to the population estimate.

However, the new version of nlmixr that is coming out doesn't require this trick (it implements it on the back-end), so you could use the etas directly.

**Reply by JonKnights** [reply]
Thanks for quick turnaround! Which estimation algorithm did you leverage? As I understand it, SAEM here ignores requests for bounds on parameter value estimates?

**Reply by mattfidler** [reply]
I tried both.  It ended up using `focei` for that  particular problem.  

However you can use `y <- logit(x , low, hi)` for the initial estimate and then in the code `par <- expit(y, low, hi)`.  This way the value is bounded by transformations.
